h/services/delete_user.py

```diff
@@ -19,38 +15,41 @@ def delete(self, user):
         """
         Deletes a user with all their group memberships and annotations.
 
-        Raises UserDeleteError when deletion fails with the appropriate error
-        message.
+        If a user owns groups with collaborators, the user is unassigned
```

Comment:
This comment is slightly ambiguous since "collaborators" could just mean that the group has other members. I would suggest stating a bit more explicitly what "collaborators" means.

Reply:
+1 Though including the word "collaborators" is helpful (just also explaining what a _collaborator_ is).